distributed/gateway/confs/deployment.toml

```diff
@@ -121,6 +121,7 @@ enable = true
 enable = false
 {{- end }}
 server_id = "{{ .Values.wso2.apim.configurations.transactionCounter.serverId }}"
+server_url = "{{ .Values.wso2.apim.configurations.transactionCounter.serverUrl }}"
```

Comment:
When transaction counter is disabled, no point is handling the server_url. Shall we fix it?